### PR TITLE
Feat/UI post

### PR DESCRIPTION
**Pull request by wikbro7650**

### CODE REVIEW

File: src/pages/FeedScreen/FeedScreen.jsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { useState, useCallback } from 'react';
+import { Children } from 'react/cjs/react.production.min';
```
Comment (T0ffik) [inline src/pages/FeedScreen/FeedScreen.jsx:2]: This shouldn't be here.

File: src/pages/FeedScreen/FeedScreen.jsx [hunk 2]
```diff
@@ -13,6 +15,14 @@ export const FeedScreenPage = () => {
             <Modal showModal={showModal} setShowModal={setShowModal}>
                 <div>Children element</div>
             </Modal>
+            <Post
+                date={new Date()}
+                name='atr'
+                avatar='https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'
+                profileID='1'
+            >
+                {Children}
```
Comment (T0ffik) [inline src/pages/FeedScreen/FeedScreen.jsx:24]: This also shouldn't be there just string like "Children element" above

**Review by T0ffik — CHANGES_REQUESTED** [review]

File: src/pages/FeedScreen/FeedScreen.jsx (hunk 2 above)
Comment (wikbro7650) [reply src/pages/FeedScreen/FeedScreen.jsx]: corrected

File: src/components/molecules/Post/Post.scss [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            grid-area: icons2;
+            margin-left: 0.5rem;
+        }
+        &__comment_button{
+            grid-area: icons3;
+        }
+    }
+}
+
+// @include for-tablet-landscape-up {
```
Comment (Veryyapeee) [inline src/components/molecules/Post/Post.scss:64]: please remove any commented code if it's non needed

File: src/components/molecules/Post/Post.scss [hunk 4]
```diff
@@ -0,0 +1,81 @@
+@import '../../../styles/mixins.scss';
+@import '../../../styles/variables.scss';
+
+.post__field {
+    margin: 1rem 3rem 0 3rem;
```
Comment (Veryyapeee) [inline src/components/molecules/Post/Post.scss:5]: Just a note: if you use flex you could write these styles like 4 times shorter, but that's just a note, i don't want you to refactor that, I don't think that we have time for that

File: src/components/molecules/Post/Post.jsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            </Text>
+            <Text size='small' customClass='date' type='primary'>
+                {dateFormat(date)}
+            </Text>
+            <Text type='primary' customClass='message' size='medium'>
+                {children}
+            </Text>
+            <Icon
+                iconName='like'
+                // onClick={}
```
Comment (Veryyapeee) [inline src/components/molecules/Post/Post.jsx:38]: remove any commented code pls

File: src/components/molecules/Post/Post.jsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
+import { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+// import { Form } from 'formik';
```
Comment (Veryyapeee) [inline src/components/molecules/Post/Post.jsx:4]: Remove

File: src/components/molecules/Post/Post.jsx [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+import { routes } from '../../../routes/Routes';
+import { Icon } from '../../atoms/IconComponent/Icon';
+import { dateFormat } from '../../../utils/format';
+
+import './Post.scss';
+
+export const Post = ({ children, date, name, avatar, profileID }) => {
+    const navigate = useNavigate();
+    const redirect = useCallback(() => {
+        navigate(routes.Profile);
```
Comment (Veryyapeee) [inline src/components/molecules/Post/Post.jsx:15]: pls add TODO comment that we need to pass an profile ID here to route

File: src/pages/FeedScreen/FeedScreen.jsx [hunk 8]
```diff
@@ -13,6 +17,19 @@ export const FeedScreenPage = () => {
...
+            <div className='post'>
+                <Post
+                    date={new Date()}
+                    name='Norbert Nowak'
+                    avatar='https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'
+                    profileID='1'
+                >
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam a blandit augue.
+                </Post>
+                <div className='form__field'>
```
Comment (Veryyapeee) [inline src/pages/FeedScreen/FeedScreen.jsx:29]: form should be a part of post component

File: src/components/molecules/Post/Post.jsx [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+            <Text type='primary' customClass='message' size='medium'>
+                {children}
+            </Text>
+            <Icon
+                iconName='like'
+                // onClick={}
+                size='medium'
+                className='like_button'
+            />
+            <Icon
```
Comment (Veryyapeee) [inline src/components/molecules/Post/Post.jsx:42]: these icons should be clickable and should change the state (filled and not) ~ this should be done via the API, but you can prepare logic for that

File: src/components/molecules/Post/Post.jsx [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+                iconName='dislike'
+                // onClick={}
+                size='medium'
+                className='dislike_button'
+            />
+            <Icon
```
Comment (Veryyapeee) [inline src/components/molecules/Post/Post.jsx:48]: Also I don't see the number of likes/dislikes and comments which I see on designs

**Review by Veryyapeee — CHANGES_REQUESTED** [review]
Some minor comments, generally looks good to me for now,

File: src/components/molecules/Post/Post.jsx (hunk 10 above)
Comment (T0ffik) [reply src/components/molecules/Post/Post.jsx:48]: Also about desings I think we missing separating line between posts. But I don't know how much we care about faithful representation of designs in current situation.

File: src/components/molecules/Post/Post.jsx (hunk 5 above)
Comment (wikbro7650) [reply src/components/molecules/Post/Post.jsx:38]: It should be there, but it should be related to the counter next to the icons for which I had no idea / time, so in order not to block it, I just left it. Finish it or do we do a separate task for it? I guess it is not that important and I don't know if we can make it

**Review by T0ffik — APPROVED** [review]

File: src/components/molecules/Post/Post.jsx (hunk 9 above)
Comment (wikbro7650) [reply src/components/molecules/Post/Post.jsx:42]: to create it we don't have to add "filled" version of icons as another component? Then, when clicked, the icon will change and the possible counter will increase

File: src/components/molecules/Post/Post.scss (hunk 3 above)
Comment (wikbro7650) [reply src/components/molecules/Post/Post.scss]: done

File: src/components/molecules/Post/Post.jsx (hunk 6 above)
Comment (wikbro7650) [reply src/components/molecules/Post/Post.jsx]: done

File: src/components/molecules/Post/Post.jsx (hunk 7 above)
Comment (wikbro7650) [reply src/components/molecules/Post/Post.jsx:14]: done

File: src/pages/FeedScreen/FeedScreen.jsx (hunk 8 above)
Comment (wikbro7650) [reply src/pages/FeedScreen/FeedScreen.jsx]: done

File: src/components/molecules/Post/Post.jsx (hunk 10 above)
Comment (wikbro7650) [reply src/components/molecules/Post/Post.jsx:47]: I added a line ;) I wrote about the numbers above

**Review by Veryyapeee — APPROVED** [review]
I guess for now it's ok, probably it would need some adjustments with real api integration but yeah, for now it's dine for me

**Pull request merged**